objects:
  april_tag_cube_23: 
    april_tag:
      goal_id:
        23
    position:
      x: 0.7
      y: 2.4
      z: 0.89
    orientation:
      x: 0.0
      y: 0.0
      z: -0.66
      w: 0.7451744
  april_tag_cube_8: 
    april_tag:
      goal_id:
        8
    position:
      x: 1
      y: 2.4
      z: 0.89
    orientation:
      x: 0.0
      y: 0.0
      z: -0.689
      w: 0.72
#you can add more products to the simulation and then add the information here
  
  april_tag_cube_24: 
    april_tag:
      goal_id:
        24
    position:
      x: 0.7
      y: 2.7
      z: 0.89
    orientation:
      x: 0.0
      y: 0.0
      z: -0.66
      w: 0.7451744
  april_tag_cube_15: 
    april_tag:
      goal_id:
        15
    position:
      x: 1
      y: 2.7
      z: 0.89
    orientation:
      x: 0.0
      y: 0.0
      z: -0.689
      w: 0.72